apps/music/js/ui/views/player_view.js

```diff
@@ -274,6 +274,7 @@ var PlayerView = {
 
   setRepeat: function pv_setRepeat(value) {
     var repeatClasses = ['repeat-off', 'repeat-list', 'repeat-song'];
+    var repeatLabels = ['repeat-off', 'repeat-list', 'repeat-song'];
```

Comment:
Sure. I was worried that this would break if someone changed the class names and didn't notice it was used for strings too.